projects/components/src/button/button.ts

```diff
@@ -3,6 +3,7 @@ export const enum ButtonRole {
   Primary = 'primary',
   Secondary = 'secondary',
   Tertiary = 'tertiary',
+  Quaternary = 'quaternary',
```

Comment:
We might want a different name for this at some point, but I can't come up with anything better right now.